locales/pt-BR/translation.json

```
@@ -90,6 +90,29 @@
         "workday-waiver-manager": "Gerenciador de Folgas",
         "yes-please": "Sim, por favor"
     },
+    "$FixedCalendar": {
+        "leave-by": "Baseado na hora que chegou hoje, você deve sair às"
+    },
+    "$FlexibleDayCalendar": {
+        "switch-view": "Trocar Visualização",
+        "current-day": "Ir para o Dia Atual",
+        "previous-day": "Dia Anterior",
+        "time-to-leave": "Hora de Ir Embora",
```

Time to Leave is the name of the app, so I think it's better to keep it without a translation:

```suggestion
        "time-to-leave": "Time to Leave",
```